web/beacon-app/src/features/onboarding/components/steps/name/index.tsx

```diff
@@ -1,8 +1,33 @@
+import { Trans } from '@lingui/macro';
+import { Button, Heading, TextField } from '@rotational/beacon-core';
+
 const nameStep = () => {
   return (
-    <div>
-      <h1>nameStep</h1>
-    </div>
+    <>
+      <Heading as="h1" className="text-lg font-bold">
+        <Trans>Step 3 of 4</Trans>
+      </Heading>
+      <p className="mt-4 font-bold">
+        <Trans>What's your name?</Trans>
+      </p>
+      <p className="my-4">
+        <Trans>
+          Adding your name will make it easier for your teammates to collaborate with you.
+        </Trans>
+      </p>
+      <div className="max-w-6xl">
+        <TextField
+          fullWidth
+          placeholder="Ex. Haley Smith"
+          label="Name"
```

Comment:
I added the label here and in step 3 for anyone who may use a screen reader since we don't have a label displayed. The `sr-only` class ensures that the label does not appear in the UI and should only be noticed by screen readers.